src/xdpdrop/src/bpf/vmlinux.h

```diff
@@ -0,0 +1 @@
+../../../vmlinux_508.h
```

Comment:
should probably point to vmlinux.h (which is a symlink to specific version). But ideally, is it possible specify location of vmlinux.h somehow without this?

Reply:
Maybe we should add a `-I` option to libbpf-rs's `cargo libbpf build` command.